companion/src/eeprominterface.h

```diff
@@ -80,16 +80,18 @@ enum Capability {
   Mixes,
   Timers,
   TimersName,
-  CustomFunctions,
-  SafetyChannelCustomFunction,
   VoicesAsNumbers,
   VoicesMaxLength,
   MultiLangVoice,
   ModelImage,
   Pots,
   Sliders,
   Switches,
+  FactoryInstalledSwitches,
   SwitchesPositions,
+  NumTrimSwitches,
+  CustomFunctions,
```

Comment:
The 2 last ones are needed here, this is good, but the 2 first ones should move to the new Board class